### PR TITLE
*/*: install {bash, fish, zsh}-completion files unconditionally

### PR DESCRIPTION
**Pull request by thesamesam**
QA policy [0] says that we don't conditionalise installation of
small files. It's a wasteful rebuild and inconsistent across packages
for when users desire completions to be available.

[0] https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301
Signed-off-by: Sam James <sam@gentoo.org>

### CODE REVIEW

File: gui-wm/sway/sway-9999.ebuild [hunk 1]
```diff
@@ -80,8 +80,8 @@ src_configure() {
 		-Dtray=$(usex tray enabled disabled)
 		-Dxwayland=$(usex X enabled disabled)
 		$(meson_use wallpapers default-wallpaper)
-		$(meson_use fish-completion fish-completions)
-		$(meson_use zsh-completion zsh-completions)
+		"-Dfish-completions=true"
+		"-Dzsh-completions=true"
 		"-Dbash-completions=true"
 		"-Dwerror=false"
```
Comment (SoapGentoo) [inline gui-wm/sway/sway-9999.ebuild:86]: remove this pointless and ugly quotes?

File: app-misc/tmux-xpanes/tmux-xpanes-4.1.2-r1.ebuild [hunk 2]
```diff
@@ -29,8 +23,7 @@ src_install() {
 	dobin bin/*
 	doman man/*.1
 	einstalldocs
-	if use zsh-completion; then
-		insinto /usr/share/zsh/site-functions
-		doins completion/zsh/*
-	fi
+
+	insinto /usr/share/zsh/site-functions
+	doins completion/zsh/*
```
Comment (SoapGentoo) [inline app-misc/tmux-xpanes/tmux-xpanes-4.1.2-r1.ebuild:28]:
```suggestion
	doins -r completion/zsh/.
```

File: app-misc/khard/khard-0.17.0-r1.ebuild [hunk 3]
```diff
@@ -40,15 +41,13 @@ DEPEND="
 
 DOCS=( CHANGES CONTRIBUTING.rst README.md doc/source/examples/khard.conf.example )
 
+python_test() {
+	esetup.py test
+}
```
Comment (SoapGentoo) [inline app-misc/khard/khard-0.17.0-r1.ebuild:46]: `distutils_enable_tests setup.py` after `*DEPEND`?

File: app-misc/khard/khard-0.13.0-r1.ebuild [hunk 4]
```diff
@@ -44,15 +42,13 @@ DEPEND="
 
 DOCS=( AUTHORS CHANGES README.md misc/khard/khard.conf.example )
 
+python_test() {
+	esetup.py test
+}
```
Comment (SoapGentoo) [inline app-misc/khard/khard-0.13.0-r1.ebuild:47]: `distutils_enable_tests setup.py` after `*DEPEND`?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

**Review by SoapGentoo — APPROVED** [review]

**Comment by thesamesam** [comment]
All done!

**Pull request closed without merging**